spec/models/petition_spec.rb

```diff
@@ -3022,4 +3122,58 @@
       I18n.with_locale(:"cy-GB") { expect(Petition.new.locale).to eq("cy-GB") }
     end
   end
+
+  describe "#create" do
+    it "sets threshold for referral to Site value" do
+      petition = FactoryBot.build(:petition)
+
+      expect {
+        petition.save
+      }.to change {
+        petition.read_attribute(:threshold_for_referral)
+      }.from(nil).to(Site.threshold_for_referral)
```

Comment:
Probably want to add an expectation in this test that the method is called, e.g.

``` ruby
expect(Site).to receive(:threshold_for_referral).and_return(123)

expect {
  petition.save
} .to change {
  petition[:threshold_for_referral]
}.from(nil).to(123)
```